src/components/MarkdownTextArea/index.jsx

```diff
@@ -0,0 +1,128 @@
+import classNames from 'classnames';
+import { string, func, number } from 'prop-types';
+import React, { Component } from 'react';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import Markdown from '../Markdown';
+
+@withStyles(theme => {
+  const borderColor =
+    theme.palette.type === 'light'
+      ? fade(theme.palette.common.black, 0.23)
+      : fade(theme.palette.common.white, 0.23);
+
+  return {
+    tab: {
+      flexGrow: 1,
+      color: theme.palette.text.primary,
```

Comment:
Added in a colour setting to the tab component since I noticed the labels would inherit from parent and become invisible:

![Screen Shot 2019-03-29 at 5 03 51 PM](https://user-images.githubusercontent.com/13473834/55262619-ba65d380-5244-11e9-9f48-08fdabf7c0ed.png)